### ISSUE TITLE
Unpackerr can't connect to Radarr

### ISSUE THREAD

**Issue by Kraowe** [davidnewhall/unpackerr#211]
I have just set up Unpackerr. It connects fine to Sonarr, but to Radarr it doesnt work. It says:
[ERROR] Radarr (http://192.168.10.40:7878): api.Get(queue): failed:  (status: 404 Not Found): invalid status code, <200||>299:

Radarr logs says:

2022-07-01 03:59:23.6|Debug|Api|[GET] /api/v3/queue/details: 200.OK (1 ms)
2022-07-01 03:59:23.6|Debug|Api|[GET] /api/v3/queue/details: 200.OK (1 ms)
2022-07-01 03:59:23.6|Debug|Api|[GET] /api/v3/queue/details: 200.OK (3 ms)
2022-07-01 03:59:23.6|Debug|Api|[GET] /api/v3/command: 200.OK (1 ms)
2022-07-01 03:59:23.6|Debug|Api|[GET] /api/v3/command: 200.OK (1 ms)
2022-07-01 03:59:23.6|Debug|Api|[GET] /api/v3/command: 200.OK (1 ms)
2022-07-01 03:59:25.2|Debug|Api|[GET] /api/queue?apikey=(removed)&order=asc&sort_by=timeleft: 404.NotFound (0 ms)
2022-07-01 03:59:27.5|Debug|Api|[GET] /api/v3/health: 200.OK (1 ms)
2022-07-01 03:59:27.5|Debug|Api|[GET] /api/v3/system/status: 200.OK (9 ms)
2022-07-01 03:59:27.7|Debug|Api|[GET] /api/v3/diskspace: 200.OK (193 ms)
2022-07-01 03:59:29.1|Debug|Api|[GET] /api/v3/log/file: 200.OK (2 ms)

And no I dont have any base url. Both sonarr and radarr only have ip:port.
- UN_SONARR_0_URL=http://192.168.10.40:8989
- UN_RADARR_0_URL=http://192.168.10.40:7878

Might be important to add, Radarr is version 4.1.0.6175

**Comment by davidnewhall** [comment]
What version is unpackerr?

**Comment by Kraowe** [comment]
> What version is unpackerr?

Thank you! It works now.

Weird, I installed Unpackerr first time today using a docker-compose file. I'm using the golift/unpackerr image. I assumed that first time it pulled the image it would pull the latest one. But I had to do docker-compose pull && docker-compose up -d and then it was updated and now Radarr works.

What I did earlier today I made my docker file and then "docker-compose up -d". Idk why it didn't pull the latest release.
